lib/jsonrpc2/interface.rb

```diff
@@ -398,10 +399,19 @@ def invoke_server_error_hook(error)
       log_error("Server error hook failed - #{error.class}: #{error.message} #{error.backtrace.join("\n    ")}")
     end
 
+    def invoke_before_validation_hook(method, id, params)
+      before_validation(method: method, id: id, params: params)
+    rescue Exception => error
```

Comment:
It may be worth catching a `StandardError` here instead
```suggestion
    rescue => error
```

https://thoughtbot.com/blog/rescue-standarderror-not-exception

Reply:
Ah, yeah. I completely forgot you mentioned this yesterday 🤦 . I was rushing to get in for dinner. 😂